in-depth/indepth-demo
# Add notifications

- [x] Personal PR review completed
- [x] Deployed and tested on Heroku
- [x] Tested on iPhone
- [x] Tested on SSR
- Add button to test notifications
- Make notification work on both desktop and android

Notes:
Should work on android but haven't been able to test due to not being https
Does not work on Safari or IOS.

This PR closes issue #49
